lib/simulator.js

```diff
@@ -36,27 +33,38 @@ async function getSimulator (udid) {
     throw new Error(`No sim found with udid ${udid}`);
   }
 
-  handleUnsupportedXcode(xcodeVersion);
-
   log.info(`Constructing iOS simulator for Xcode version ${xcodeVersion.versionString} ` +
            `with udid '${udid}'`);
-  if (xcodeVersion.major === 6) {
-    return new SimulatorXcode6(udid, xcodeVersion);
-  } else if (xcodeVersion.major >= 7 && xcodeVersion.major < 8) {
-    if (xcodeVersion.minor < 3) {
-      return new SimulatorXcode7(udid, xcodeVersion);
-    } else {
-      return new SimulatorXcode73(udid, xcodeVersion);
-    }
-  } else if (xcodeVersion.major === 8) {
-    return new SimulatorXcode8(udid, xcodeVersion);
-  } else if (xcodeVersion.major === 9) {
-    if (xcodeVersion.minor < 3) {
-      return new SimulatorXcode9(udid, xcodeVersion);
-    } else {
-      return new SimulatorXcode93(udid, xcodeVersion);
-    }
+  let SimClass;
```

Comment:
it's a bit confusing to start local variable names with uppercase char

Reply:
Is it? It will be holding a class. It's also assigned within a couple of lines. I don't find it confusing at all.

Reply:
i agree with @imurchie on this one, it even has `Class` in the variable name which is what makes it not confusing to me.